disk-utils/fdisk.c

```diff
@@ -535,7 +535,7 @@ void list_partition_types(struct fdisk_context *cxt)
 						name, sizeof(name),
 						&width, MBS_ALIGN_LEFT, 0);
 
-				if (ret == (size_t)-1 || ret >= sizeof(name))
+				if (ret == SIZE_MAX || ret >= sizeof(name))
 					printf("%-15.15s",
```

Comment:
(size_t) -1 is pretty common patters (see for example libc mbrtowc())  and it's more readable than SIZE_MAX.

Reply:
I dropped this change from pull request.